engine/src/main/java/io/camunda/zeebe/engine/state/deployment/VersionManager.java

```diff
@@ -160,4 +154,6 @@ public Optional<Integer> findResourceVersionBefore(
     idKey.wrapString(resourceId);
     return getVersionInfo().findVersionBefore(version);
   }
+
+  private record TenantIdAndProcessId(String tenantId, String processId) {}
```

Comment:
❌ As the `VersionManager` is used also by Forms, this record should be renamed

```suggestion
  private record TenantIdAndResourceId(String tenantId, String resourceId) {}
```

Reply:
Good catch 👍